src/main.ts

```diff
@@ -22,11 +23,6 @@ async function bootstrap() {
 	});
 	
 	const config = new DocumentBuilder()
-		.addApiKey({
```

Comment:
Was this apiKey never used?

Reply:
No this was just copy paste from the example